src/themes/brutalist.css

```
@@ -0,0 +1,832 @@
+    --border-width: var(--wa-panel-border-width);
+    --border-color: var(--wa-color-surface-border);
+    --content-color: var(--wa-color-base-10);
+    text-transform: uppercase;
+
+    &::part(base) {
+      padding: var(--wa-space-xs) var(--wa-space-s);
+    }
+
+    &[variant='brand'] {
+      --background: var(--wa-color-blue-90);
+    }
+    &[variant='success'] {
+      --background: var(--wa-color-green-90);
+    }
+    &[variant='warning'] {
+      --background: var(--wa-color-yellow-90);
+    }
+    &[variant='danger'] {
+      --background: var(--wa-color-red-90);
+    }
+    &[variant='neutral'] {
+      --background: var(--wa-color-base-90);
+    }
+  }
+
+  wa-tag {
+    --border-width: 0 0 0 var(--wa-panel-border-width);
+
+    &[variant='brand'] {
+      --border-color: var(--wa-color-brand-border-subtle);
+    }
+    &[variant='success'] {
+      --border-color: var(--wa-color-success-border-subtle);
+    }
+    &[variant='warning'] {
+      --border-color: var(--wa-color-warning-border-subtle);
+    }
+    &[variant='danger'] {
+      --border-color: var(--wa-color-danger-border-subtle);
+    }
+    &[variant='neutral'] {
+      --border-color: var(--wa-color-neutral-border-subtle);
+    }
+  }
+
+  wa-tag::part(base) {
+    padding: var(--wa-space-m);
+  }
+
+  wa-input,
+  wa-select,
+  wa-checkbox {
+    --border-color: var (--wa-color-neutral-border-highlight);
+  }
+
+  /* preventing shadows on all form input types */
+  wa-input:not(:focus),
+  wa-input:not([filled]):not([disabled]),
+  wa-select:not([filled]):not([disabled]) {
+    --box-shadow: var(--wa-shadow-level-0);
+    --border-width: 0 0 var(--wa-form-controls-border-width) var(--wa-form-controls-border-width);
+  }
+
+  .wa-button-group__button:not(.wa-button-group__button--first) {
+    margin-inline-start: calc(-1 * var(--wa-form-controls-border-width));
+  }
+
+  wa-radio-button {
+    background: var(--wa-color-surface-default);
+  }
+
+  wa-menu {
+    border-width: var(--wa-form-controls-border-width);
+  }
+
+  wa-switch {
+    --thumb-color: var(--wa-form-controls-activated-color);
+    --thumb-color-checked: var(--wa-color-surface-default);
+    --background: transparent;
+    --background-checked: var(--wa-form-controls-activated-color);
+    --border-color: var(--wa-form-controls-activated-color);
+    /* --border-color-checked: var(--background-checked); */
+    --height: calc(1.2em * var(--wa-font-line-height-compact) - 0.125rem);
+  }
+
+  wa-switch::part(control),
+  wa-switch::part(thumb) {
+    border-radius: var(--wa-corners-base);
+    --wa-font-line-height-compact: 5px;
+  }
+
+  wa-switch::part(thumb) {
+    --border-width: 0;
+    box-shadow: inset 0 0 0 3px var(--wa-color-surface-default);
+  }
+  wa-switch[checked]::part(thumb) {
+    box-shadow: inset 0 0 0 3px var(--background-checked);
+  }
+
+  .preview-container {
+    background: var(--wa-color-base-95) url('/assets/images/kitchen-sink/brutalist/hero.png') 50% 3% no-repeat;
```

```suggestion
    background: var(--wa-color-surface-lowered) url('/assets/images/kitchen-sink/brutalist/hero.png') 50% 3% no-repeat;
```

This (plus an adjustment to dark mode's `--wa-color-surface-lowered` below) means that we don't have to specifically override the `background-color` for `.preview-container` in dark mode.